ui/ducks/send/send.js

```diff
@@ -2188,24 +2245,36 @@ export function updateSendQuote(
       draftTransaction?.sendAsset?.details?.address !==
       draftTransaction?.receiveAsset?.details?.address;
 
+    const {
+      quotes,
+      swapQuotesError,
+      isSwapQuoteLoading,
+      swapQuotesLatestRequestTimestamp,
+    } = draftTransaction;
```

Comment:
I think we need to either return early if there's no draft transaction, or add a fallback if draft tx is undefined like this: 

```suggestion
    const {
      quotes,
      swapQuotesError,
      isSwapQuoteLoading,
      swapQuotesLatestRequestTimestamp,
    } = draftTransaction ?? {};
```

Reply:
There should be a `draftTransaction` anytime the user is one the send flow (see: `initializeSendState`):

```
    if (!draftTransaction) {
      return thunkApi.rejectWithValue(
        'draftTransaction not found, possibly not on send flow',
      );
    }

Reply:
Doesn't hurt to have it though so we can add the object fallback.